README.md

```diff
@@ -17,22 +36,58 @@ keytool -genkey -alias selfsigned -keyalg RSA -keystore src/main/docker/etc/conf
 **Make sure the key password and store password are the same!** This is a requirement for Spring Security.
 
 2. Then, make sure [Docker][] and [Docker-Compose][] are installed on your system.
-3. Finally, we can start the stack with `docker-compose -f src/main/docker/app.yml up -d`.
+3. Finally, we can start the stack with `docker-compose -f src/main/docker/management-portal.yml up -d`.
 
 The docker image can be pulled by running `docker pull radarcns/management-portal:0.3.1`.
 
 ## Configuration
 
-First create a keypair to sign JWT with:
-
-```shell
-keytool -genkey -alias selfsigned -keyalg RSA -keystore src/main/resources/config/keystore.jks -keysize 4048 -storepass radarbase
-```
-
-To add a new client to this API, add it to `changelogs/config/liquibase/oauth_client_details.csv`.
+Management Portal comes with a set of default values for its configuration. These defaults are most
+easily overridden by using environment variables.
+
+### Environment Variables
+
+The table below lists the variables that are most likely in need of change when deploying Management
+Portal. You can find the complete configuration
+in the [application.yml](src/main/resources/config/application.yml) and 
+[application-prod.yml](src/main/resources/config/application-prod.yml) files. See
+[Spring external configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html)
+for other options on overriding the default configuration.
+
+| Variable                                                   | Default value                                       | Description                                                                                            |
+|------------------------------------------------------------|-----------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| `SPRING_DATASOURCE_URL`                                    | `jdbc:postgresql://localhost:5432/managementportal` | URL for the database to be used                                                                        |
```

Comment:
This should be moved under docker section. Configuring environment.properties is adequate for standard config